.github/workflows/pr.yml

```diff
@@ -43,7 +43,7 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x, 16.x]
```

Comment:
if we've updated to build with 16, why also use 14 here?

Reply:
Took this from our chatter:
The reason is to minimize switching environments forth and back to different types of versions while working on a few projects and switching between them.